users/details/utils.js

```diff
@@ -36,3 +36,169 @@ function generateNoDataFoundSection(message) {
   const container = document.querySelector('.user-details-header');
   container.appendChild(notFoundDiv);
 }
+
+function dateDiff(date1, date2, formatter) {
```

Comment:
I think this function is used in other places also. Can you move this to the common utils and import it from there for both pages instead of duplicating the code?

Reply:
Done